github/pulls.go

```diff
@@ -28,49 +28,63 @@ type PullRequestAutoMerge struct {
-	ReviewCommentURL    *string               `json:"review_comment_url,omitempty"`
-	ReviewComments      *int                  `json:"review_comments,omitempty"`
-	Assignee            *User                 `json:"assignee,omitempty"`
-	Assignees           []*User               `json:"assignees,omitempty"`
-	Milestone           *Milestone            `json:"milestone,omitempty"`
-	MaintainerCanModify *bool                 `json:"maintainer_can_modify,omitempty"`
-	AuthorAssociation   *string               `json:"author_association,omitempty"`
-	NodeID              *string               `json:"node_id,omitempty"`
-	RequestedReviewers  []*User               `json:"requested_reviewers,omitempty"`
-	AutoMerge           *PullRequestAutoMerge `json:"auto_merge,omitempty"`
+	ID                 *int64                `json:"id,omitempty"`
+	Number             *int                  `json:"number,omitempty"`
+	State              *string               `json:"state,omitempty"`
+	Locked             *bool                 `json:"locked,omitempty"`
+	Title              *string               `json:"title,omitempty"`
+	Body               *string               `json:"body,omitempty"`
+	CreatedAt          *Timestamp            `json:"created_at,omitempty"`
+	UpdatedAt          *Timestamp            `json:"updated_at,omitempty"`
+	ClosedAt           *Timestamp            `json:"closed_at,omitempty"`
+	MergedAt           *Timestamp            `json:"merged_at,omitempty"`
+	Labels             []*Label              `json:"labels,omitempty"`
+	User               *User                 `json:"user,omitempty"`
+	Draft              *bool                 `json:"draft,omitempty"`
+	URL                *string               `json:"url,omitempty"`
+	HTMLURL            *string               `json:"html_url,omitempty"`
+	IssueURL           *string               `json:"issue_url,omitempty"`
+	StatusesURL        *string               `json:"statuses_url,omitempty"`
+	DiffURL            *string               `json:"diff_url,omitempty"`
+	PatchURL           *string               `json:"patch_url,omitempty"`
+	CommitsURL         *string               `json:"commits_url,omitempty"`
+	CommentsURL        *string               `json:"comments_url,omitempty"`
+	ReviewCommentsURL  *string               `json:"review_comments_url,omitempty"`
+	ReviewCommentURL   *string               `json:"review_comment_url,omitempty"`
+	Assignee           *User                 `json:"assignee,omitempty"`
+	Assignees          []*User               `json:"assignees,omitempty"`
+	Milestone          *Milestone            `json:"milestone,omitempty"`
+	AuthorAssociation  *string               `json:"author_association,omitempty"`
+	NodeID             *string               `json:"node_id,omitempty"`
+	RequestedReviewers []*User               `json:"requested_reviewers,omitempty"`
+	AutoMerge          *PullRequestAutoMerge `json:"auto_merge,omitempty"`
+
+	// Merged is not populated by List operation.
```

Comment:
It is unnecessary and overly verbose to individually list each field's name separately.
Instead, please simply make this a block with a single comment at the top, like this:

```suggestion
	// These fields are not populated by the List operation.
```